### PR TITLE
Add Babel transpilation step to handle default export

### PR DESCRIPTION
**Pull request by vsashyn**

### CODE REVIEW

**Comment by kof** [comment]
lgtm, if it works I will merge it right away without testing, please confirm you tested it)

**Comment by vsashyn** [comment]
Tested it on my local machine. Works well for me 😄 The one thing I thought about, is users who used **repl** and set previous code example in `localStorage` with `... return { }`. This example will no longer work. I'm not sure how to fix it. If you have any thoughts about it, please let me know 😄

**Comment by kof** [comment]
We should add logic which resets the localStorage on each new release. For e.g. we can add repl version to the key of local storage, so that new version doesn't use the old thing

**Comment by vsashyn** [comment]
Got it. Will add in this PR

File: src/index.js [hunk 1]
```diff
@@ -68,15 +69,16 @@ function evalModule(str) {
 }
 
 function load({input, output}) {
-  const jssStr = localStorage.jss || example
+  const jssStr = localStorage[pkginfo.version] || example
```
Comment (kof) [inline src/index.js]: I would add something like `jss-repl-${pkginfo.version}` so that the key in local storage is not tooo generic

**Comment by kof** [comment]
merged!

**Pull request merged**